app/components/Admin/Room/RoomDetail.jsx

```diff
@@ -90,47 +76,48 @@ export default ({state}) => {
         <div>
             <h3>Room Detail</h3>
             <div>
-                <form>
-                    <div className="form-group row">
-                        <label htmlFor="key" className="col-2 col-form-label">Key</label>
-                        <div className="col-10">
-                            <label htmlFor="key" className="col-10 col-form-label">{tempRoom.key}</label>
+                <form className="form-horizontal">
+                    <div className="form-group">
+                        <label htmlFor="key" className="control-label col-sm-2">Key</label>
```

Comment:
Some inputs are too small on medium and lower screens (can only see 1 number), maybe something closer to this html, you can figure out how you like it of course. Just check that it looks ok on different screens. There is a "mobile" button in the console of Chrome, top left button, it's good for checking all different screen sizes
```html
  <div className="form-group col-xs-12 col-sm-6 col-md-4">
    <label htmlFor="name" className="control-label">Room Name</label>
    <input className="form-control" type="text" required value={tempRoom.name} id="name" name="name"/>
  </div>
```